sky/packages/sky_engine/BUILD.gn

```diff
@@ -110,6 +110,17 @@ copy("internal") {
       [ "$root_gen_dir/dart-pkg/sky_engine/lib/internal/{{source_file_part}}" ]
 }
 
+copy("vm_internal") {
```

Comment:
Can we make this target work like the other targts where we only need to depend on a gn thingy?

Reply:
Yes, that is an alternative and the downside(from https://github.com/flutter/flutter/issues/128880#issuecomment-1908978582) is that it could get out of date.